src/components/sbb-alert/sbb-alert.tsx

```diff
@@ -185,69 +180,75 @@ export class SbbAlert implements ComponentInterface, LinkProperties {
     };
   }
 
-  public render(): JSX.Element {
-    return (
+  protected override render(): TemplateResult {
+    return html`
       <div
         class="sbb-alert__transition-wrapper"
-        ref={(el): void => {
-          this._transitionWrapperElement = el;
-        }}
+        ${ref((el): void => {
+          this._transitionWrapperElement = el as HTMLElement;
+        })}
```

Comment:
instead of "as", I'd prefer to write: (Also for every other occurrences in the render() function.)

```suggestion
        ${ref((el: HTMLElement): void => {
          this._transitionWrapperElement = el;
        })}
```